pages/guides/auth/client-credentials-flow.mdx

```diff
@@ -69,9 +69,12 @@ application_token = $dwolla.auths.client
 ```
 
 ```php
-/**
- *  No support for this language yet. We recommend using an external REST client for making OAuth requests.
- **/
+<?php
+// Using dwollav2 - https://github.com/Dwolla/dwolla-swagger-php
```

Comment:
Should this comment be "Using dwollaswagger" ?

Reply:
Ah yeah. This [commit](https://github.com/Dwolla/dev-portal/pull/172/commits/56c5d2158b1600e93d82aa9217ff78767adaee31) should fix it! 👍